src/styles/main.scss

```diff
@@ -1,3 +1,57 @@
-body {
-  margin: 0;
+$daySize: 98px;
+$dayOfWeek: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
+$monthLengths: 28, 29, 30, 31;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 708px;
+  gap: 1px;
+  padding: 10px;
+
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+  &__day {
```

Comment:
please check out [this link](https://github.com/necolas/idiomatic-css#declaration-order) and pay attention to properties' declaration order.
fix it in entire file